python/lsst/ts/electrometer/controller.py

```diff
@@ -238,11 +237,14 @@ async def disconnect(self) -> None:
         self.image_service_client = None
         await self.commander.disconnect()
 
-    async def perform_zero_calibration(self) -> None:
+    async def perform_zero_calibration(self, mode, auto, set_range):
```

Comment:
So I think the best course of action is to set any class variables here and not in their other functions (e.g. `set_range`).
I believe it simplifies the code to have it all in one place.